### PR TITLE
Cluster resource browser bugfixes

### PR DESCRIPTION
**Pull request by vivek-devtron**
# Description

This PR contains the changes for bug fixes in cluster resource browser merge

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/css/base.scss [hunk 1]
```diff
@@ -2815,6 +2815,10 @@ textarea,
     height: 32px !important;
 }
 
+.h-35 {
```
Comment (shivani170) [inline src/css/base.scss:2818]: Is it okay to use h-35? Can't we go with h-36?

File: src/components/common/DynamicTabs/DynamicTabs.tsx [hunk 2]
```diff
@@ -40,15 +40,15 @@ function DynamicTabs({ tabs, removeTabByIdentifier, stopTabByIdentifier, enableS
         initTabsData(tabs, setTabsData, setSelectedTab, closeMenu)
     }, [tabs])
 
-    useEffect(() => {
-      if (enableShortCut) {
-          shortcut.registerShortcut(handleInputShortcut, ['t'], 'TabSearchFocus', 'Focus tab search')
-      }
+  //   useEffect(() => {
```
Comment (shivani170) [inline src/components/common/DynamicTabs/DynamicTabs.tsx:43]: please remove commented code

File: src/components/common/DynamicTabs/DynamicTabs.component.tsx [hunk 3]
```diff
@@ -22,10 +22,10 @@ export const MoreButtonWrapper = ({ children, isMenuOpen, onClose, toggleMenu, t
             ref={tabPopupMenuRef}
         >
             <DropDown
-                className={`rotate icon-dim-20 ml-2 mr-2 pointer ${isMenuOpen ? 'fcn-9' : 'fcn-7'}`}
+                className={`rotate icon-dim-20 p-2 pointer ${isMenuOpen ? 'fcn-9' : 'fcn-7'}`}
                 style={{ ['--rotateBy' as any]: isMenuOpen ? '90deg' : '-90deg' }}
             />
-            <span className='flex icon-dim-24 cn-7 fs-12 fw-6 dc__border-left bc-n50'>t</span>
+            {/* <span className='flex icon-dim-24 cn-7 fs-12 fw-6 dc__border-left bc-n50'>t</span> */}
```
Comment (shivani170) [inline src/components/common/DynamicTabs/DynamicTabs.component.tsx:28]: Please remove commented code

File: src/components/ResourceBrowser/ResourceBrowser.service.tsx [hunk 4]
```diff
@@ -32,6 +32,10 @@ export const getResourceGroupList = (clusterId: string, signal?: AbortSignal): P
     })
 }
 
+export const getResourceGroupListRaw = (clusterId: string): Promise<APIResourceResponse> => {
```
Comment (shivani170) [inline src/components/ResourceBrowser/ResourceBrowser.service.tsx:35]: typo error `getResourceGroupListRow`

Comment (vivek-devtron) [reply src/components/ResourceBrowser/ResourceBrowser.service.tsx:35]: It is as expected

File: src/components/common/DynamicTabs/DynamicTabs.tsx (hunk 2 above)
Comment (vivek-devtron) [reply src/components/common/DynamicTabs/DynamicTabs.tsx]: Done

File: src/components/common/DynamicTabs/DynamicTabs.tsx [hunk 5]
```diff
@@ -83,8 +83,8 @@ function DynamicTabs({ tabs, removeTabByIdentifier, stopTabByIdentifier, enableS
                 <div
                     className={`flex left ${isSelected ? 'cn-9' : ''} ${isDeleted ? 'dynamic-tab__deleted cr-5' : ''}`}
                 >
-                    {iconPath && <img className="icon-dim-16 mr-8" src={iconPath} alt={name} />}
-                    {_showNameOnSelect && <span className="fs-12 fw-6 lh-20 dc__ellipsis-right" data-testid={name}>
+                    {iconPath && <img className="icon-dim-16" src={iconPath} alt={name} />}
+                    {_showNameOnSelect && <span className={`fs-12 fw-6 lh-20 dc__ellipsis-right ${iconPath?'ml-8': ''} `} data-testid={name}>
```
[the remark this answers is not in the document]
Comment (vivek-devtron) [reply src/components/common/DynamicTabs/DynamicTabs.tsx]: Done

**Review by shivani170 — APPROVED** [review]

**Pull request merged**